unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/FriendsHUD/Scripts/FriendsHUDComponentView.cs

```diff
@@ -0,0 +1,230 @@
+        remove => friendRequestsTab.OnFriendRequestApproved -= value;
+    }
+    public event Action<FriendRequestEntry> OnCancelConfirmation
+    {
+        add => friendRequestsTab.OnCancelConfirmation += value;
+        remove => friendRequestsTab.OnCancelConfirmation -= value;
+    }
+    public event Action<FriendRequestEntry> OnRejectConfirmation
+    {
+        add => friendRequestsTab.OnRejectConfirmation += value;
+        remove => friendRequestsTab.OnRejectConfirmation -= value;
+    }
+    public event Action<string> OnFriendRequestSent
+    {
+        add => friendRequestsTab.OnFriendRequestSent += value;
+        remove => friendRequestsTab.OnFriendRequestSent -= value;
+    }
+    public event Action<FriendEntry> OnWhisper
+    {
+        add => friendsTab.OnWhisper += value;
+        remove => friendsTab.OnWhisper -= value;
+    }
+    public event Action<string> OnDeleteConfirmation
+    {
+        add => friendsTab.OnDeleteConfirmation += value;
+        remove => friendsTab.OnDeleteConfirmation -= value;
+    }
+    public event Action OnClose;
+
+    public RectTransform Transform => transform as RectTransform;
+    
+    public static FriendsHUDComponentView Create()
+    {
+        var view = Instantiate(Resources.Load<GameObject>("SocialBarV1/FriendsHUD")).GetComponent<FriendsHUDComponentView>();
+        return view;
+    }
+
+    public override void Awake()
+    {
+        base.Awake();
+        
+        friendsTabFocusButton.onClick.AddListener(() => FocusTab(0));
+        friendRequestsTabFocusButton.onClick.AddListener(() => FocusTab(1));
+        closeButton.onClick.AddListener(() =>
+        {
+            OnClose?.Invoke();
+            Hide();
+        });
+        
+        friendsTab.Expand();
+        friendRequestsTab.Expand();
+    }
+
+    public void HideSpinner()
+    {
+        loadingSpinner.SetActive(false);
+        model.isLoadingSpinnerActive = false;
+    }
+
+    public void ShowSpinner()
+    {
+        loadingSpinner.SetActive(true);
+        model.isLoadingSpinnerActive = true;
+    }
+
+    public List<FriendEntryBase> GetAllEntries()
+    {
+        var result = new List<FriendEntryBase>();
+        result.AddRange(friendsTab.Entries.Values);
+        result.AddRange(friendRequestsTab.Entries.Values);
+        return result;
+    }
+
+    public FriendEntryBase GetEntry(string userId)
+    {
+        return (FriendEntryBase) friendsTab.Get(userId) ?? friendRequestsTab.Get(userId);
+    }
+
+    public void UpdateEntry(string userId, FriendEntryBase.Model model)
+    {
+        friendsTab.Populate(userId, model);
+        friendRequestsTab.Populate(userId, model);
+    }
+
+    public void DisplayFriendUserNotFound() => friendRequestsTab.ShowUserNotFoundNotification();
+
+    public bool IsFriendListFocused() => friendsTab.gameObject.activeInHierarchy;
+
+    public bool IsFriendListCreationReady() => friendsTab.DidDeferredCreationCompleted;
```

Comment:
Only as suggestion: As the returned data is a simple boolean, wouldn't it be better to declare these functions (`IsFriendListFocused()` and `IsFriendListCreationReady`) as properties? It is the same from the technical point of view, but maybe the code is a bit cleaner.